include/boost/sml.hpp

```diff
@@ -480,6 +488,14 @@ struct zero_wrapper<R (TBase::*)(TArgs...) const noexcept> {
  private:
   R (TBase::*ptr)(TArgs...) const noexcept {};
 };
+template <class R, class... TArgs>
+struct zero_wrapper<R (*)(TArgs...) noexcept> {
+  explicit zero_wrapper(R (*ptr)(TArgs...) noexcept) : ptr{ptr} {}
```

Comment:
Oh, that will require a guard (__has_noexcept or whatever is called)
* https://travis-ci.org/github/boost-ext/sml/jobs/728458162

Reply:
It's already guarded by the `#if defined(__cpp_noexcept_function_type)` at line 474.